### ISSUE TITLE
Compression ration lower than pigz

### ISSUE THREAD

**Issue by necros2k7** [sstadick/crabz#15]
pigz -n -b 64 -k -p 31 -11 sample.file
crabz -l 9 sample.file -o sample.gz

not much but still

**Comment by sstadick** [comment]
Thanks for making an issue! There are two issues with your comparison. First `crabz` uses `zlib-ng` backend by default whereas `pigz` uses `zlib`. Second `pigz` uses a different algorithm when you go over [compression level 9](https://github.com/madler/pigz/blob/cb8a432c91a1dbaee896cd1ad90be62e5d82d452/pigz.c#L4240). Which `crabz` does not support.

**Comment by necros2k7** [comment]
So can you perhaps beat pigz in future?)

**Comment by sstadick** [comment]
True! Want to change the title of this issue to something like "Add zopfli compression format"? That is what pigz is using when `-11` is specified. It trades off a _lot_ of speed for a modest gain in compression. You could also try compressing with `mgzip` format and compression turned way up (11 or 12 I think). It _might_ get close.
